docker/gaffer-integration-tests/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:7
```

Comment:
We should be using jdk 8 and it's probably useful to use a slim image to reduce size.